scripts/kind-e2e/e2e.sh

```diff
@@ -187,6 +187,8 @@ with_context cluster2 connectivity_tests
 
 # dataplane E2E need to be modified for globalnet
 if [[ $globalnet = false ]]; then
+    unset KUBECONFIG
```

Comment:
This is related to pipefail? Or just some random cleanup? (which is fine, but wanted to make sure you meant to do it here:)

Reply:
Indeed it is, please look at the specific commit to understand why :) - https://github.com/submariner-io/submariner-operator/pull/317/commits/c1d27c71b460eb0248814e75aaa9b511bc159139

Reply:
Ah I see your explanation, thanks!